opentelemetry-sdk/src/logs/log_emitter.rs

```diff
@@ -140,14 +151,38 @@ impl LoggerProvider {
 struct LoggerProviderInner {
     processors: Vec<Box<dyn LogProcessor>>,
     resource: Resource,
+    is_shutdown: AtomicBool,
 }
 
 impl Drop for LoggerProviderInner {
     fn drop(&mut self) {
-        for processor in &mut self.processors {
-            if let Err(err) = processor.shutdown() {
-                global::handle_error(err);
+        if self
```

Comment:
I agree with this change, but please make it own separate PR, so we can keep this PR strictly for replacing global error handler with internal logger.

Reply:
This was done in response to the comment - https://github.com/open-telemetry/opentelemetry-rust/pull/2184#discussion_r1793842371. Should we keep it in this PR with separate changelog entry?

Reply:
I always prefer short, very focused PRs! 
One can easily review them and continuously make progress by merging short PRs quickly. Slow and steady!